src/components/ListItem.js

```diff
@@ -11,22 +12,26 @@ import ListItemEditForm from './ListItemEditForm';
 import ListItemControls from './ListItemControls';
 
 const spanSource = {
+  // When focused on item disable dragging
+  canDrag(props) {
+    return props.canDrag;
+  },
```

Comment:
Similar to `setCanDrag`, can you explain in a comment why this is necessary instead of just using `props.canDrag`?